DataFormats/HcalRecHit/interface/HFQIE10Info.h

```diff
@@ -21,6 +21,14 @@ class HFQIE10Info
     static const unsigned N_RAW_MAX = 5;
     static const raw_type INVALID_RAW = std::numeric_limits<raw_type>::max();
 
+    // Special value for the rise time used in case the QIE10 pulse
+    // is always below the discriminator
+    static constexpr float UNKNOWN_T_UNDERSHOOT = -100.f;
+
+    // Special value for the rise time used in case the QIE10 pulse
+    // is always above the discriminator
+    static constexpr float UNKNOWN_T_OVERSHOOT = -101.f;
```

Comment:
These are just technical numbers. The only requirements
for them is that they are exactly representable floats, and
that they can not be encountered during normal readouts.
It is also nice to define them in such a way that, if desired,
they can be seen in plots together with "normal" readouts.

On 06/19/2016 11:17 AM, Slava Krutelyov wrote:
>
> In DataFormats/HcalRecHit/interface/HFQIE10Info.h <https://github.com/cms-sw/cmssw/pull/14920#discussion_r67619481>:
>
> > @@ -21,6 +21,14 @@ class HFQIE10Info
> >      static const unsigned N_RAW_MAX = 5;
> >      static const raw_type INVALID_RAW = std::numeric_limits<raw_type>::max();
> >
> > +    // Special value for the rise time used in case the QIE10 pulse
> > +    // is always below the discriminator
> > +    static constexpr float UNKNOWN_T_UNDERSHOOT = -100.f;
> > +
> > +    // Special value for the rise time used in case the QIE10 pulse
> > +    // is always above the discriminator
> > +    static constexpr float UNKNOWN_T_OVERSHOOT = -101.f;
>
> overshoot is smaller than undershoot?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/cms-sw/cmssw/pull/14920/files/0b14bc4b41a8878079e1ed29025e58c078e6f68a#r67619481>, or mute the thread 
> <https://github.com/notifications/unsubscribe/AFS4-Ypcq6z48vojFrUR7sY5QkcmfNaEks5qNWu0gaJpZM4I5Mg7>.
>